changelog/unreleased/kong/plugins-add-standard-webhooks.yml

```diff
@@ -0,0 +1,4 @@
+message: |
+  Add standard webhooks plugin
```

Comment:
Please state what function does the plugin provide.

Reply:
the function is "standard webhooks", see https://www.standardwebhooks.com/